tests/test_bevel.py

```diff
@@ -27,7 +27,7 @@ def fancy_edge(tmp_path: Path, edge_type: str, part_no: str):
     sheet.render("freecad")
 
 
-def test_edge_round(tmp_path: Path):
+def q_test_edge_round(tmp_path: Path):
```

Comment:
These test match against an exact hash of a binary file. But we have found that the files contain timestamps, thus this is not a good comparison. Disabling tests, rather have tests we trust.